src/main/java/teammates/storage/entity/Student.java

```diff
@@ -12,12 +12,11 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * An association class that represents the association Account -->
- * [enrolled in] --> Course.
+ * An association class that represents the association Account --> [enrolled
+ * in] --> Course.
```

Comment:
The original has better readability. Don't follow the line length cap blindly (it is a soft limit only). Break line line where the readability is retained while avoiding 'too long' lines. The suggested line break by the IDE may not be the optimal.